### ISSUE TITLE
The Issue Chat Room LoL

### ISSUE THREAD

**Issue by jewbmx** [jewbmx/repo#52]
This is a general chit chat spot for whoever lmao.
Since imma be cleaning up the account soon i figured id place another one of these for yall.

(BTW i was gonna make a lazy tool for advancedsettings.xml but now if you rock kodi v21 or higher the good settings are included so there is no need to really use advanced settings anymore :p lol so if ya havent updated kodi lately ya should.)

**Comment by jewbmx** [comment]
If ya dont see it elsewhere, expect a new update for scrubs to come around the 4th of july. Could probably be sooner but its "scheduled" like this to give folks that help out time to contribute whatever before i start lol.
Also as a warning to those that help, i might go ahead and recode all the scrapers and the scrape_sources module if i get bored so beware lol altho it might solve some folks issues i been seeing. Besides the obvious ignorant issue people cry about which is a normal lack of sources lmao.

**Comment by jewbmx** [comment]
Ive decided to ditch yomovies due to it mostly being hindi shit and not worth the time to tweak it to be english only lol.
If anyone wants to keep using it you can save it now and while your at it you could also pull 3 domains from it by going thru the old ones i saved in it. Should be 2 that goto one, and another 2 that goto one, then one more leading to the final working domain. But one of the 3 domains are coded different so you gotta fix the scraper to match it lol. (I pulled em then made em into 3 separate working scrapers before realizing that i didnt ever plan on keeping the trash in the first place lol)

**Comment by jewbmx** [comment]
If anyone is interested in assisting lol can i get someone to tell me if primewire_mx is broken or working, it appears dead on my end. Then another is epxmovies_net, seems dead to me also but i have 3 alternatives to test in that scraper but cant actually do it myself as the sites dont work for me lmao. They are epxmovieshd.net, europixhd.site, and topgmovies.net  (id suggest building urls out of these domains and adding the scrapers search_link to em in order to test em but whatever works lol)
Imma probably come across a few more scrapers that need a little outside help so i will likely post em here when i come across em.
Also fair warning that imma probably be slower then i originally planned so dont be surprised if my update is delayed longer lol im too high and lazy lately with little ambition and alot more work needed done then i expected lol

**Comment by Melann956** [comment]
Hi. Both primewire and epx, ephmovieshd are out. They are not working on my end either. Europix and Topgmovies site is running. ✌️

Melissa

________________________________

**Comment by jewbmx** [comment]
Thanks for the speedy help lol, the rest of that stuff can be left to someone that tinkers in the scrapers and wants to bother with em.
Imma probably try to finish going thru the current scraper pile tomorrow then post any others that are blocked for me. Might go ahead and keep that in mind for when i start digging into my new url list too incase someone wants to make a scraper out of whatever i cant lol

**Comment by Melann956** [comment]
These 2 sites are up and running. https://onionplay.se and
http://www.streamlord.com.

Melissa

________________________________

**Comment by jayshomebrew** [comment]
Same results
epxmovies.net DOWN  
topgmovies.net UP
europixhd.site UP

**Comment by jewbmx** [comment]
You wanna try sticking those sites into the epxmovies scraper to see if they will work then? Lol.
And thanks again melissa that will probably save me some time

**Comment by jewbmx** [comment]
Dont recall if the gomovies_sx scraper ever worked but i just threw it away because it was annoying lol. If it did work or is wanted its gonna need someone else tinkering with it because its a buzz kill in my eyes lmao

**Comment by Melann956** [comment]
How would I stick them in the scraper? Not as tekky as you. Lol Oh and gomovies never worked.

Melissa

________________________________

**Comment by jayshomebrew** [comment]
I did modify epxmovies_net.py to get it to work; however, topgmovies.net and europixhd.site are both just embed sites, so nothing new here.  
<a href="https://imgur.com/1KKLaVE.png">
  <img src="https://imgur.com/1KKLaVE.png" />
</a>

I did get gomovies_sx to work for a while, but yes probably not worth the effort. 

If you get a chance, take a look at [goojara.py](https://github.com/jayshomebrew/ScraperWork/blob/main/temp/goojara_to.py) as its still working for me.

**Comment by Melann956** [comment]
Hi. Yes goojara works for me as well. So far it's upmovies.filelions upmovies.upstream and goojara.wootly that mainly work with them. I know to choose those specifically because they work. Before almost all of upmovies worked but now it's mainly those. Hope this helps.

Melissa

________________________________

**Comment by jewbmx** [comment]
Yea i pretty much chucked all those sites as they are basically worthless for me same with a couple scrapers here and there lol.
As for goojara its working on my end but i havent began the host end of my update yet so the actual sources mean nothing to me at the moment but i plan to do some work there and add some new ideas/features for user customization and whatnot.

As for your question melissa, what i was aiming for there was basically just someone that can open the sites that fail on my end who could also open the scrapers in a notepad and edit the base_url and domains to be those new domains/sites and run the scraper or scrapers in a test run to see if any of them new sites could be used lol. It doesnt really matter to me tho since i cant use them scrapers or sites on my end so its sorta just me tossing the work off on someone that might benefit from it lol.

Also for a status update incase anyones wondering, my update progress is horribly lacking lol. Ive barely gotten anywhere with the current scraper pile and have yet to start making more new scrapers besides a few new ones i have tossed into a temp folder which includes a couple from everyone while i was "gone" lol. My guess is that i got a week or more of time before i finish unless i get some random ambition soon lmao

**Comment by jewbmx** [comment]
here is a list of the scrapers that i find to be new duds, if anyone wants to test them and see if they work for you i will add them back to the working folder for ya to use.
i think they are all in the addon already and in the working folder since i havent touched the old folder labeled duds yet lol.


123series_expert.py
anymovies_co.py
cinehub_wtf.py
epxmovies_net.py
flixhq_click.py
flixtor_expert.py
genvideos_org.py
goku_to.py
gowatchseries_ch.py
lordhd_one.py
m4ufree_com.py
movies123_space.py
musichq_expert.py
primewire_mx.py
putlocker_to.py
ssoap2day_to.py
thebestcinema_online.py
vumoo_to.py
watcha_movie.py
winnoise_com.py
yifymovies9_pro.py

I also have about 28 other scrapers that i didnt list here which seem to not be working for me but im going to tinker with each one and find out why they are not working before ditching any of em or adding em to a duds list lol.
then afterwards imma start going thru my new url list and making scrapers from it before moving on to the source/host issues and other features or problems.

**Comment by jewbmx** [comment]
Im also ditching the scrapers that fail and use my search_engine module like tubitv because its too annoying to try and get that shit to work lol, i added 2 decent options as new defs and they both dont seem to actually work in kodi so i gave up on em all lmao. Honestly never cared for them anyways because search engines are always gay.

**Comment by Melann956** [comment]
Gotcha! ✌️

Melissa
________________________________

**Comment by Melann956** [comment]
Does your Gmail still work? Sent ya...

Melissa
________________________________

**Comment by jewbmx** [comment]
Yea i replied to ya lol

**Comment by Melann956** [comment]
Links that are working are goojara, upmovies and couchturner. So far they're pulling alot of links. If anyone is wondering..

**Comment by jewbmx** [comment]
Supposedly my net provider was gonna be ditching this annoying McAfee security feature that cock blocks most the streaming sites but im starting to think they lied or that the fuckers are gonna make me have to buy another modem that way it will be gone lol. Im so tired of waiting around 😒

**Comment by jayshomebrew** [comment]
I did go through your list above and had the same results. 
So what I do is find a cheap VPS, setup a personal VPN, then your ISP nor any site won't block you. Its not the most efficient; but it works. Its probably a little bit more expensive than a VPN, but you get more control.  Check out the forums on lowendbox and spend some time shopping for a while, as there is usually deals for like $25/year for 3 years, for like 3GB mem/20TB bandwidth.  Setting up the VPN is pretty easy as you can use dietpi [debian].

**Comment by jewbmx** [comment]
Well i could most likely do all my work with a vpn and get away with it by using a free option too, but thats just a bit more effort that i dont wanna bother with lol. My plan is to pretty much take my time going thru my big ass site list to make some new scrapers out of and save anything i dont use. Then hopefully this isp protection craps change will kick in by the end of the month so i can finally be free lmao. But if that dont happen i will just leave it be for a while and share my stuff that way others can try to tinker with whatever. Eventually i will have to do something tho because the blockage is getting pretty bad.
My biggest issue tho is that im so bored and ran out of anime/toons to watch so i been rewatching all the good ones like reincarnated rpg type of ones that have been pretty popular the past few years lol

**Comment by jayshomebrew** [comment]
Please send me one so I can take a crack at it.
truck-kun is all the rage.

**Comment by jewbmx** [comment]
if your wanting a scraper to tinker with you can have this one lol  https://github.com/jewbmx/ScraperWork/blob/main/temp/levidia_ch.py

it doesnt work right now and likely needs something added to it like whats used in the current goojara_to scraper. everything seems fine till you touch the source links and the process is flawless in a browser so id guess the cookie shit and so on just needs added.

if thats not what ya wanted then you gotta be more specific lol.

**Comment by jewbmx** [comment]
I need me a simple site that will let me input a url press a button and have the page source shown lol, this isp cockblocking is so damn obnoxious

**Comment by ghost** [comment]
> I need me a simple site that will let me input a url press a button and have the page source shown lol, this isp cockblocking is so damn obnoxious

Not sure if these sites will help;

https://www.view-page-source.com

https://viewsourcepage.com

**Comment by jewbmx** [comment]
Lol ill have to check those out and likely find one myself if i get into it enough. Used to have a good one that was part of a big ass web tools site but the site died off.

As for my url list i was goin thru, i went ahead and uploaded it a min ago with the 2 scrapers i didnt finish incase anyone wants to toy with em. Note that the url list has anime/toon urls that i didnt touch for scrubs and also has a shit ton of clone urls that are basically saved there incase their scraper counterparts die off.(likely already have 3 of each scraper now and thats my assumed limit lol.)
And for the "trashbin list" thats towards the bottom its basically everything i bothered not to delete but didnt use or load/open lol.

If anyone wants to make something from this url list tho they can probably toss it back to me and have it thrown into the addon.
As for the anime/toon stuff its pretty much saved for folks like the otaku addon people and possibly for the day that i finally make a little custom indexer or light addon for em lol.(been a plan thats never happened in all my years of coding lmao.)

**Comment by jayshomebrew** [comment]
[9pm_to](https://github.com/jayshomebrew/ScraperWork/blob/main/temp/9pm_to.py) ready to test.

**Comment by jewbmx** [comment]
Nice its working fine lol altho i just tried one movie and closed kodi i assume the rest is fine too.
Should be handy to have since all the scraper work hasnt really increased the results and the hosts still seem to be a bit slim.
I still got alot of work to do with the scrape_sources module aka host work but after that imma likely push the update. If you wanna make any others or improve anything id guess about a week to do so before my update. 
As of now ive pretty much watched every good anime atleast twice so imma be forced back into real movies and shows soon lol

**Comment by jewbmx** [comment]
If anyone thats handy at scraper work and not blocked by my kinda isp issues, you can probably check out the dud folders in the addon to see if a few are working again. Id suggest logging lines/steps on whatever has a active site then fixing whatever might be needed. Altho you could also just toss them all into the working folder and see if any of em return results like i do when im lazy lmao.

Imma go ahead and push my update in a day or two and probably toss the rest of my todo list for a later date like some of the host work and ideas i have for setting changes. Which means if anyone wants to add their scraper contributions before i update they dont got long left or they can get in on the next update lol.
That includes the levidia_ch scraper that i tossed into a temp folder labeled finish, right now it seems to still need work for the sources after go.php like goojara_to uses. But since its basically a clone of that site i dont really care about finishing it so its on the back burner lol.

**Comment by jayshomebrew** [comment]
Ready to test [levidia_ch](https://github.com/jayshomebrew/ScraperWork/blob/main/temp/levidia_ch.py); but similar to goojara as you said.    Some of the dood.watch links do work.

**Comment by jewbmx** [comment]
Damn you really finished that one already lmao. All ive done is get high, eat burgers, drink my coffee and watch naruto while occasionally searching the net for various anime addons to try or checkout lol.
Which gave me a smart idea, otaku has the same feature ive seen a few times which is a db logged record of your last watched content which would likely be nice in scrubs instead of having to work your way thru the menu and loading times like workin your way back thru a big ass trakt progress list or hunting down that userlist you dug the show out of lmao. The process is pretty basic and everything needed is basically already in the addon so i just gotta work out the bits and splice some code into a working feature that will hopefully function exactly how i want it lol.
Im also planning to eventually reorganize the scraper settings to be sorted/grouped by site/code style then alpha instead of the basic alpha we have now, that way its easier to toggle off the dupes we dont really need to use as i kinda try to limit them to 3 dupes itd still be cool to easily toggle them dupes off for a quicker cleaner search. Then theres some similar type of changes for the scrape_sources module and the hosts used in the addon and whatnot.

**Comment by jayshomebrew** [comment]
bstsrs_one is now [bstsrs_in](https://github.com/jayshomebrew/ScraperWork/blob/main/temp/bstsrs_in.py) and works for me.

**Comment by jewbmx** [comment]
crap i forgot to say what was already fixed so you ended up having to touch a couple ive already done like bstsrs, altho i think you added some nice code to that one for the starting url that wasnt there before and pretty wise to have lol.
as for levidia_ch i will run it now and toss it in its place, was also thinking of moving the wootly code to the scrape_sources module since it just seems to be a special host resolver that could be reused with ease when them links show up in the future.
regardless of all this tho id suggest you chill for now and i will push my update for you today. then we can do whatever we wanna after the addon updates and just push another update later on.

**Comment by jewbmx** [comment]
updates pushed plus a little repo cleaning, ill probably do the rest of my github cleanup at some other time because im way too high right now lol.
you can now have the current scraper folders tho and whatnot so if you decide to tinker with the scrapers and such it will hopefully be easier and all that.
my thoughts are pretty much a little more scraper work then moving on to the sources/results links and hosts which moves to the scrape_source module updates then as another end result cleaning up the scraperwork folder/repo on here. then eventually all them other plans or ideas i got lol but yall can likely help out with all this stuff thats not locked in my mind if ya wanna like anything that my net cockblocks since its unlikely that i will get to it.
oh and as a side note there was one random scraper i found a 2embed result on for movies but i didnt notice which one of the duds it was and since i cant use 2embed i just ignored it and tossed the duds pile back into its folder while also turning my 2 duds folders into one lol.

**Comment by Melann956** [comment]
Thanks @jewbmx!!! 😊

**Comment by jewbmx** [comment]
Im looking into some ideas and got myself wondering something, do any of yall use simkl like trakt and if so do you use it for more then anime lol.  If your curious it can be used the same as tmdb and trakt plus has pretty much every feature they do so adding its content and features would be pretty beneficial besides knowing if the account end of their code/api would be used or useful lol

**Comment by jayshomebrew** [comment]
`use simkl like trakt? ` Only trakt
`If simkl, do you use it for more anime or shows? ` N/A
`would try simkl ?`   Yes, unsure of the benefits.

**Comment by jewbmx** [comment]
If you have confusion just goto simkl.com and browse the site a little. Click the movies section and see some of their categories then you can click lists to see a nice pile of random lists lol its a decent supply for movies and shows along with its anime stuff that i think is its most used for. But basically if i add it all into scrubs just based on its movies and shows content that will greatly improve the discovery aspect of the addon due to the wide variety of different stuff lol. And the fact that i love chasing custom userlists of movies and whatnot totally doesn't impact things lmao

**Comment by jewbmx** [comment]
My idea for the simkl usage is pretty much like opening any of the scrubs menu sections already available, if i recall it goes like "movies" then "explore ..." with everything related to that area of content piled in and separated by suppliers like tmdb. So id basically just add a new section for simkl and pile its stuff into there lol and perhaps continue onward the same way i did with tmdb by adding some of its trailer stuff and account stuff.
But i think my decision will be finalized after i already make most the code and run some tests, because i dont actually wanna add the anime stuff in at all and if it cant be done that way imma ditch the idea lol. I just gotta tinker a bit and figure things out lol. Also one downfall is that it might make me look into the watched marking stuff and i dont wanna venture there lmao.

**Comment by jewbmx** [comment]
So it looks like adding simkl is a waste of time now unless people actually use simkl accounts lol.
Their api is easy to make yet has like a 1000 daily limit so if i added it into scrubs everyone would probably need to make their own personal api or end up with errors from reaching the rate limit.
The site says there is a way to get rid of the limit but im not sure how thats done so o well lol. Think imma give up on the idea but perhaps also save it for another day incase their api ever changes for the better.

**Comment by jewbmx** [comment]
My brain wont leave me alone so im probably gonna be adding some simkl parts to scrubs anyways and maybe add more later.
Think im also gonna go ahead and start making a anime/toon addon since my mind keeps drifting towards the shit, plus i cant help but wanna rape all these sites i find but have no current use for lol.
So yall will probably need to tell me about scrubs issues incase i get too deep the anime shit and if anyone wants to assist in the new addon im happy to pass it around when it gets going. As for the coding its likely just gonna be a copy of scrubs gutted and rebuilt similar to how the json_iptv addon was done mixed with how me and tempest did that one temptv addon. Which means the essential codebase will likely be about the same and probably wont be very confusing lol.

**Comment by jewbmx** [comment]
You know your high when you spend 3 hours recoding the anime api your working on repeatedly because you think there is a error, even tho the real issue is that you just forgot to toggle the debug setting back on.(which was totally toggled off during a random test with another setting lol)

**Comment by Melann956** [comment]
Lol


________________________________

**Comment by jewbmx** [comment]
Anyone bored enough to try making a api module for simkl?
I think i have most of it in a file and ready to be made, but i dont really understand the user end of the api lol. As of right now i can probably get the watchlist and whatnot but i havent figured out if its the right way to do it. I also havent figured out the other basic things needed like adding a item to a list and removing a item lol. Thinking thats something important to have so im kinda stuck in limbo at the moment. 
The goal is to code a module that basically has everything their api offers and to have content type separate so i can do whatever with just tv shows or movies or even anime.

Since there doesnt seem to be any "modules or wrappers" available anywhere its probably gonna take me a while to figure it all out lol

**Comment by rrosajp** [comment]
@jewbmx 
long time user of your work thanks for all you do.
i thinks there was SIMKL module in Umbrella here it is
but it only accessing simkl_list but 

[https://raw.githubusercontent.com/umbrellaplug/umbrellaplug.github.io/refs/heads/master/omega/plugin.video.umbrella/resources/lib/modules/simkl.py](url)

**Comment by jewbmx** [comment]
@rrosajp
(Just to copy ya lol)
Long time no see :) and thanks for the link, i can use that as a little guide just like otaku while i try figuring this api out lol. Regardless of what it has it will atleast be helpful for seeing how other folks use the api in the tricky spots that confuse me.

**Comment by jewbmx** [comment]
Got another project bored people can contribute to if they wanna lol.
Its for a furure about/help feature imma add in.
The basics are simple and dont need much for coding skills.
For each setting and option in the addon there will pretty much be a file with their id or name or tag in it followed by a little description lol.
Like...
'addon.blah': 'Enabling blah allows blah to blah.'
Which would later be added in and once the about(blah) is selected a little popup box lets ya read the description details lol.

Since its a long strenuous road to complete this little idea i thought id offer it up for yall then compare and combine any results later as i go :). Granted imma be working on another addon and doing the same with it so this wont be started for scrubs for a while still lmao.

**Comment by jewbmx** [comment]
Im glad i waited till now to code the simkl api module imma be using lol, just remembered about **kwargs and how it works so now all my defs are pretty and clean lmao. So imma be building the url params on the fly passed as kwargs then the kwargs are passed to requests as params=kwargs :)
Also recently got simkl to add the genres endpoint for movies so my final code will look proper to my ocd.
Sort of a shame that i put so much effort into something that turned out so well and is likely to never get used lol.
Now i just gotta work on the little user features like adding items to the lists and the removal bit which use some odd object structures you send as json or some shit. But it should be somewhat easy since it looks similar to trakt.

**Comment by jewbmx** [comment]
Got a user feedback type of question for ya since i cant make up my own mind lol.
Heres the run down lol...
Been using the Otaku addon for a while and got used to some things like for instance removing search items one by one instead of the entire search history. 
So i decided to go ahead and add the feature since all the needed code was basically there but needing thrown together to work in that fashion.
Now its been coded quickly and tested with no issues lol but im stuck on the final step.
Shall we make the page refresh using code or by manual movements? Pretty sure the addon used to do this type of coded refresh all over but if i recall i removed most of it or made it able to be disabled lol not really sure but do remember finding it annoying alot.

So to sum it all up im asking yall if you'd rather have the search list be refreshed automatically after using this new feature to remove a item lol.
(Note that no response to this will result in the code staying as it is with a simple removal and no refresh unless you back out the page and load it again lol.)

**Comment by jewbmx** [comment]
Guess i dont need any response about this search stuff, i got bored and just threw in a setting for it so i can forget about it lol.

**Comment by Melann956** [comment]
Hi. I was gonna ask about the episodes widget setting. I can only see the episodes when I switch it to calender. Oh and I don't mind clearing the search manually. Thanks for this awesome addon! ✌️

Melissa

________________________________

**Comment by jewbmx** [comment]
Guessing your using it with the tvmaze parts instead of trakt, i dont ever see that stuff so i will look into it and figure out whats up. If it worked before like the last update you can try tweaking the settings for cache or cache timeout, i think they are in the devish tab. I recently found out that those changes i made make the trakt progress list and watchlist run differently forcing me to use the cleaning widget menu in the "right click" menu often lol.
Also as for the search stuff you can now just "right click" a item and click the new removal option along with the new setting for it to auto refresh if ya want. Its already done and coded in so your stuck with it :p i just gotta finish some other things and get to pushing the update. Altho that might take a little longer because im pretty sure i need to try fixing a couple scrapers that return alot of sources with wrong episodes or even seasons but i also gotta track em down as well because im always too baked to remember the names once the right episode starts playing.

**Comment by Melann956** [comment]
Yep, I use tvmaze parts. I don't use trakt. Usually we'll just pick another link if it's the same episode again. As for not remembering the names take a pic of your screen. Lol

Melissa
________________________________

**Comment by jewbmx** [comment]
Lol yea i was sorta thinking id try starting a habit of logging my issues in a notepad on my phone, tried pics a couple times but i always forget em then sit around later trying to guess why the random pics were taken lol.
Oh and i went ahead and fixed that tvmaze feature for ya, it looks like some custom error bits that were meant to block trash now assume its all trash lol, apparently they made some changes somewhere in the api results or something. So instead of bothering with it all i just went the lazy route and removed that bit of code that caused the issues from use so it will load everything without the ghetto custom filtering, altho i did leave the non english bit in since its still functioning properly.
Think imma just leave it this way too and let users hit me up to suggest what they want filtered out from now on since i dont recall how it was before the issue began and dont use the tvmaze sections either.

**Comment by Melann956** [comment]
Thank you!! ✌️

Melissa

________________________________

**Comment by jewbmx** [comment]
From what i seen in the few tests i was running, imma likely have to add a custom list of show types to block like
lst = ['news', 'and', 'whatnot']
Then a comparison type of line to block em. But that depends on them user responses on what should be blocked out lol

**Comment by jewbmx** [comment]
Decided to add a useragent choice setting in the addon to fix a scraper issue which lets ya try to match the useragent to your device but only included my 3 devices lol.
Thought id say so here and see if anyone wants to add any to the current list.
I included pc(windows), phone(android), and firestick(linux).
Probably need a few others like xbox and whatnot. 
Basically ya just gotta name the device and operating system aka platform. Or in other words just a easy label for the device and the operating system label is usually in the useragent. So ya open whatever browser thats available on the device and google like "whats my useragent" and copy that useragent to a notepad with your chosen device label.
Mine as a example are just copied from a similar search using Firefox on my windows 10 laptop and Samsung Galaxy S9+ then my firesticks Silk browser. 
Devices i know of that could be useful to add for sure are the xbox and apple then maybe some other firestick type of devices.
Also i should mention that if we get enough current useragents to match all the various platforms recognized in the platform code we could make this option be automatic but i dont really wanna work that hard so im not planning to go that route unless it gets rolling on its own lol.
Oh and if anyone is curious on why id rather use real current useragents compared to the usual route of generating em or using em from the usual lists around the net its because matching it to your real devices useragent is a better way to go with the trickier sites that wanna cock block, and should help hide the kodi requests compared to kodi requests usually looking like some "robot requests" lol.

**Comment by ghost** [comment]
> Decided to add a useragent choice setting in the addon to fix a scraper issue which lets ya try to match the useragent to your device but only included my 3 devices lol. Thought id say so here and see if anyone wants to add any to the current list. I included pc(windows), phone(android), and firestick(linux). Probably need a few others like xbox and whatnot. Basically ya just gotta name the device and operating system aka platform. Or in other words just a easy label for the device and the operating system label is usually in the useragent. So ya open whatever browser thats available on the device and google like "whats my useragent" and copy that useragent to a notepad with your chosen device label. Mine as a example are just copied from a similar search using Firefox on my windows 10 laptop and Samsung Galaxy S9+ then my firesticks Silk browser. Devices i know of that could be useful to add for sure are the xbox and apple then maybe some other firestick type of devices. Also i should mention that if we get enough current useragents to match all the various platforms recognized in the platform code we could make this option be automatic but i dont really wanna work that hard so im not planning to go that route unless it gets rolling on its own lol. Oh and if anyone is curious on why id rather use real current useragents compared to the usual route of generating em or using em from the usual lists around the net its because matching it to your real devices useragent is a better way to go with the trickier sites that wanna cock block, and should help hide the kodi requests compared to kodi requests usually looking like some "robot requests" lol.

Sounds awesome to me!! 😀😃🙂🙃😊

**Comment by jayshomebrew** [comment]
> Decided to add a useragent choice setting in the addon to fix a scraper issue which lets ya try to match the useragent to your device ..

Take a look at venomous' [use of useragent](https://github.com/VeNoMouS/cloudscraper/tree/master/cloudscraper/user_agent) as its a nice implementation.  Or just grab the [browsers.json](https://github.com/VeNoMouS/cloudscraper/blob/master/cloudscraper/user_agent/browsers.json)

**Comment by jewbmx** [comment]
Yea thats one of the sources for a collection along with some useragent based sites i have thats pretty much a log of various peoples useragents saved in it that used some site lol.
Im just a little lazy and dont wanna do the research and building of my full idea lmao but probably will eventually so auto can be the first choice instead.

**Comment by rrosajp** [comment]
@jewbmx 
in control.py there is platform() function put this in there and we can use this 
auto mated one.

```
def get_useragent():
	get_platform = platform()
	# log(f'get_platform: {get_platform}', 1)
	if get_platform == 'windows': return 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/114.0'
	elif get_platform == 'linux': return  'Mozilla/5.0 (X11; Linux i686) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.94 Safari/537.36'
	elif get_platform == 'android': return 'Mozilla/5.0 (Android 10; Mobile; rv:83.0) Gecko/83.0 Firefox/83.0'
	elif get_platform == 'ios': return  'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:61.0) Gecko/20100101 Firefox/61.0',
	else: return 'Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:64.0) Gecko/20100101 Firefox/64.0'
```

**Comment by jewbmx** [comment]
Im kinda curious about the platform bit, if ya look in our control module/script our platform codes pretty basic, then if you like look at the cloudscraper's help module/script it uses the actual platform module/script thats built in and easily used yet ive rarely seen it used anywhere lol.
As for yalls code shares imma probably use a little from both plus my own idea all mixed together. The cloudscraper "module" could still be useful if like it was gutted a bit since the cloudflare stuff is now obsolete and maybe the big ass useragent file was hosted online with a one time run to cache the right ua for ya lol.

**Comment by jewbmx** [comment]
Guess there is also a third option thats like sys.platform lol. Damn multiple choice got me stuck 😐

**Comment by jewbmx** [comment]
Took the time to remake both control.platform and that get_useragent with all sorts of most likely not needed bits lol but now that im done im not sure if its wise to apply it due to the thought of the changes effecting things badly or causing issues lmao. Guessing imma run alot of tests to see how it goes, which will probably be slow since im also at the final stages of the simkl script i was making and dont wanna finish it because of all the testing i need to do for code checking and final tweaks lol.
On the plus side of this conundrum im thinking of adding a option to open a source link in a browser to play as a backup option. Maybe even the scraper page for the result and/or adding them links to the clipboard as a lazy copy n paste option.

**Comment by jewbmx** [comment]
Thought Id share this bit of news i got in a email with yall lol...

Starting on March 4, 2025, an OAuth access_token will expire in 24 hours, instead of 3 months. Since you've created a Trakt API app, we wanted to let you know directly.

(Emails From Trakt lol)

Since i will likely get high and forget this change is coming, i hope someone will let me know if the addon has issues with trakt after their change is applied :).

**Comment by Melann956** [comment]
Lol! I don't use trakt so I'm good. ✌️🙃


________________________________

**Comment by Cukipik** [comment]
Hi there Jew, love your work and trully appreciate everything your doing.

I've encounter some problems, I get links but since yesterday, almost all of them are now saying that they can play multiple files or something like that at the same time ehile they use to work just fine before. 

I did a clean install 2 weeks ago of Kodi and Scrubs V2. I was hoping you might have an idea of whats happening. Is it on my side or something change somewhere else?

Thanks for everything again.

**Comment by jayshomebrew** [comment]
Please test:  [levidia_ch.py](https://github.com/jayshomebrew/ScraperWork/blob/main/temp/levidia_ch.py) Updated to fix wootly links.  

![Image](https://github.com/user-attachments/assets/b022f8f8-b249-4cd7-b19f-dec59b58001b)

**Comment by jewbmx** [comment]
@jayshomebrew  if you supply your fix to gujal for resolveurl it might fix the links for everyone without needing to change out scraper files. If the scraper works fine to pull the links then resolveurl should be able to resolve em for ya.

**Comment by jayshomebrew** [comment]
I finally gave an update to fix resolveurl, and gujal made it even better: updated in [5.1.165](https://github.com/Gujal00/ResolveURL/blob/master/script.module.resolveurl/changelog.txt).

I was desperate, so I made a quick fix for [projectfreetv_cyou.py](https://github.com/jayshomebrew/ScraperWork/blob/main/temp/projectfreetv_cyou.py) and [lol.py](https://github.com/jayshomebrew/ScraperWork/blob/main/temp/projectfreetv_lol.py). Still, only filelions and streamwish would work for me; but enough to limp me along.

**Comment by ghost** [comment]
If anybody wants to test the following scrapers, and also just so you know I'm not a coder and I just used @jewbmx template he has provided in his repository

[Link To Scrapers](https://github.com/TempR3p0/Testing)

**Comment by jayshomebrew** [comment]
> If anybody wants to test the following scrapers, and also just so you know I'm not a coder and I just used [@jewbmx](https://github.com/jewbmx) template he has provided in his repository
> 
> [Link To Scrapers](https://github.com/TempR3p0/Testing)

Since most of these are dubbed in French/German/Hindi, I would fork scrubsv2 and make an addon yourself.

**Comment by ghost** [comment]
> > If anybody wants to test the following scrapers, and also just so you know I'm not a coder and I just used [@jewbmx](https://github.com/jewbmx) template he has provided in his repository
> > [Link To Scrapers](https://github.com/TempR3p0/Testing)
> 
> Since most of these are dubbed in French/German/Hindi, I would fork scrubsv2 and make an addon yourself.

Ah I didn't realise that, I thought they had english links 😞

**Comment by jewbmx** [comment]
If they work for you then just toss them into the scrubs addon that you have on your device so you can use em, and then remember to save them in a backup folder somewhere else like in your temp repo you shared here. This way you can enjoy using them now and when i update the addon in the future you can still toss them back in to use them again. 
I will go thru them as well and maybe some will be added into the next update. 
Also thanks for being one of the few smart folks that can make their own scrapers and contribute lol. (There is not many people with that mindset.)

**Comment by ghost** [comment]
> If they work for you then just toss them into the scrubs addon that you have on your device so you can use em, and then remember to save them in a backup folder somewhere else like in your temp repo you shared here. This way you can enjoy using them now and when i update the addon in the future you can still toss them back in to use them again. I will go thru them as well and maybe some will be added into the next update. Also thanks for being one of the few smart folks that can make their own scrapers and contribute lol. (There is not many people with that mindset.)

Thanks for the compliment ☺️  Just trying to help you out and the folks that use this awesome app you have created 😀 

Oh and looking forward to your update 😀


Update........
Removed repo till I update, as some not working on my end 😞

**Comment by jewbmx** [comment]
If you have a list of urls you can upload it with em like i do sometimes, that way other folks can try to make a scraper out of em lol.
I already downloaded a copy of your temp repo and started to look at a few so i will just compare the old one with the new one later.

**Comment by ghost** [comment]
> If you have a list of urls you can upload it with em like i do sometimes, that way other folks can try to make a scraper out of em lol. I already downloaded a copy of your temp repo and started to look at a few so i will just compare the old one with the new one later.

Sorry its taken so long, but I lost access to my GitHub account. Anyway here is the updated repo (I  know that some are French/German/Hindi, but they do have english versions on them)

[Link To Repo](https://github.com/TempR3po/Repo)

**Comment by Melann956** [comment]
@jewbmx soooo now alot of the links that did work, don't anymore. Looking forward to your update.

**Comment by jayshomebrew** [comment]
doodstream, luluvdo and filelions links should work; try this: [projectfreetv_lol.py](https://github.com/jayshomebrew/ScraperWork/blob/main/temp/projectfreetv_lol.py)
[bstsrs_one.py](https://github.com/jayshomebrew/ScraperWork/blob/main/temp/bstsrs_one.py) is back online; and working for me. 

![Image](https://github.com/user-attachments/assets/4da0e522-e2f3-49ba-b228-f0a67c4768b0)

**Comment by ghost** [comment]
I've added the scrapers I made into the addon located here if anyone wants to try it?

[ScrubsV2 - 5.1.42a](https://github.com/TempR3po/Repo)

Hope you don't mind me creating this to help you out @jewbmx

**Comment by jayshomebrew** [comment]
@jewbmx  Take a look at the resolve for [projectfreetv_lol.py](https://github.com/jayshomebrew/ScraperWork/blob/main/temp/projectfreetv_lol.py) as I got a few of them to work (resolveURL has been updated to fix a few of these).  Let me know if you have any scrapers you want me to take on.

**Comment by jewbmx** [comment]
I started to look at it but decided to save it for later and just go thru each scraper one at a time to double check every step/line lol.
As for that exact one tho i did notice that the open link urls didnt work for me by replacing it with open site, so it might need coded from now on to use both url types depending on what i find out with all the other scrapers using that system. 

Also as a heads up for yall, from my little work so far ive found plenty of sources that just need updating in the scrape_sources module like various hoster domains have changed and whatnot. Like vidsrc for example. 
If you want a simple site to look at that seems to have some variations like that which need ran thru the scrape_sources module you can check out this one i was toying with earlier called flix-wave.me  (you can toss it into the scrapers movies4fun_net or pressplay_top to try it out and log the result lines, altho the search bit needs tweaked it will still error to the hand-made/hard-coded urls so itd be fine lol.)

**Comment by jayshomebrew** [comment]
Can you post your updated scrape_sources module ?

**Comment by jewbmx** [comment]
Pretty sure mine is still stock if your asking me lol.

If anyone wants to help out yall can share any scrapers ya got/find, and can help modify whatever needs it in the scrape_sources module, along with simply sharing lists of sites lol.

**Comment by jewbmx** [comment]
If yall dont know already, the scraper process of pretty much all of em is to have source links ran thru the scrape_sources module first before building the final results list we see as a source list so most the host based tweaks can be done there instead of the resolve def in a scraper. Meaning any tweaks we make for like dood or whatnot can be used for all the scrapers instead of just the one scraper it might be placed in like some of the stuff in that custom projectfreetv_lol scraper as an example or making something work as a source that takes more page loads like 2embed and whatnot that are "embed hosters".
Also might wanna notice the host domain swapping bit which i think i have at the top of the module where like dood.xyz is swapped to dood.com, that bit of code is meant to swap the dood hosts that dont work in my location into one of the domains that do work for me. So yall can do like i do and open the resolveurl resolver for the host and manually check all the domains in a browser to make the list of failures and the list of working good ones then add em to the appropriate spot in the module so they will play for ya. (Side note/tip for ya is to log a working link for said host when you do this process so you can use that link to test all the different domains. A valid link is far more useful then a basic domain page attempt lol.)

**Comment by jewbmx** [comment]
From my current progress in updating the scrapers i can be sure that our biggest problem is the links found by the scrapers need looked at because most of em are now all embed holders so they need logged and looked at then likely new defs made for em lol counting updating the ones already made in the scrape_sources module like 2embed and vidsrc.
If anybody has enough knowledge and skill to help, you can use the log line i placed near the end of the prepare_link def to get a quick list altho it also comes with its own problems which is noted with the log line lol. (First def located near the top of the scrape_sources module.)

As for the update timeline its gonna probably take longer then imagined with how much more needs done lol

**Comment by jewbmx** [comment]
Ive gone thru the projectfreetv scrapers and found that _lol and _cyou are using a updated code system that i dont know how to crack yet so they are essentially duds now. Luckily the third scraper is still functioning on the old code system so its still a working scraper, which is watchseries_cyou altho i think it did need its base_url swapped to the previous domain thats 1.fun lmao. Also seems we need to either remove the first iframe bit of code or make a resolve function for that source since it seems to always be the same. (Granted i didnt see any reason to keep the second iframe code bit at all which is in the resolve def lol. Guessing thats from an even older code system they had.)

**Comment by jayshomebrew** [comment]
yeah the projectfree scrapers were really good.. up until recently.   Hopefully the one.fun stays around for a while.

**Comment by jewbmx** [comment]
I thought of a code change to that scraper that might help keep it around a bit longer but didnt try it out yet. Use your url replace bit to swap the open/link bit to open/site then right afterwards use the request geturl bit to snag the final links. Doing so should work fine for that 1.fun site and will reduce the impact we have on the site. But id suggest not sharing these changes if you like that scraper lol everyone will have it when i push the update. Oh and your changes in the resolve def might need modified if you use my changes, personally havent touched em yet so dont know if they will be the same lol.

Right now im checking each scraper one by one start to finish then logging a couple test results sources and saving the sources i log inside the scrapers to go thru them afterwards when i begin the resolve checking and all the embed host work lol

**Comment by ghost** [comment]
It's taking longer than i thought to look at my fix for my scrapers. Currently re writing them to only work in python 3.11 (kodi 21+) that is if i can figure it out lol

**Comment by jayshomebrew** [comment]
@jewbmx  Not sure if you got to this one, but I cleaned it all up and got it working again.  [goojara_to](https://github.com/jayshomebrew/ScraperWork/blob/main/temp/goojara_to.py)

what do you think about adding the orange text if debug?  line 914 [sources](https://github.com/jayshomebrew/ScraperWork/blob/main/temp/sources.py)

![Image](https://github.com/user-attachments/assets/5768a3a7-cdb6-470c-9890-72e5bee90134)


..and show url to doubleline if debug: line 473 [sources](https://github.com/jayshomebrew/ScraperWork/blob/main/temp/sources.py)

![Image](https://github.com/user-attachments/assets/0836c825-0733-4912-8fcc-c42633d4e0c7)

**Comment by jewbmx** [comment]
Think i already fixed that one too but i will double check later tonight and compare yours to mine anyways just incase.
As for your idea ive had a few thoughts like that as well but never decided on a idea to do so was thinking of adding multiple ideas with a setting to enable em. The downfall to it all is that everyone wanted quality crap so the source page lines have always be sorta devoted to that shit lol. My ideas for it vary from pretty much all the stuff i log in my scraper tests log lines used in like a scraper based info/results menu widget like clicking information and seeing all the various result info for a source and its scraper and options to like copy the various urls to clipboard or open in a browser lol. Then also using various ideas of changing the source pages design to be stuff i wanna see with custom options to swap each bit out the way id wanna while allowing others to do theirs in whatever way they want too.
To be honest the addons so old now that id feel bad for others if i just went and changed stuff up in a forceful manner so my idea for customized settings stuff is golden lol but at the same time i like keeping the settings simple and light.
Ive also been thinking about doing a new addon made from scratch so i can make every little thing the way i want but thats a slow project thats been ignored since the whole python 3 changes in kodi lmao

**Comment by jewbmx** [comment]
I was thinking of the wrong scraper lol, i went ahead and tossed in your version and probably just gonna clean it up a little since it seems to work fine.
As for its vidsrc sources if you decide to make them work id suggest making the def for em in scrape_sources because there are more scrapers that return them links. (I havent got that far yet.)

**Comment by jewbmx** [comment]
Im gonna tone down my ocd and leave all my log lines in the scrapers lol its annoying to redo em every update so imma say screw it and just comment them out and sacrifice the little bits they take up on the harddrive lol.

Oh and i finished touching that goojara_to scraper now so i can toss it back to ya if you wanna do a final check or look at it jayshomebrew. (Altho as a warning i saved some source results in the bottom of it like the other scrapers ive completed so its technically still a bit unfinished in that regard lol.)

**Comment by jayshomebrew** [comment]
> I was thinking of the wrong scraper lol, i went ahead and tossed in your version and probably just gonna clean it up a little since it seems to work fine. As for its vidsrc sources if you decide to make them work id suggest making the def for em in scrape_sources because there are more scrapers that return them links. (I havent got that far yet.)

@jewbmx  please add for vidsrc sources:  [scrape_sources](https://github.com/jayshomebrew/ScraperWork/blob/main/temp/scrape_sources_vidsrc_only.py)

I'm not sure if I shared bstsrs_one w/ you; as it has a new [decoder](https://github.com/jayshomebrew/ScraperWork/blob/main/temp/bstsrs_one.py). 

Is scrubsv2 pulling in all the domains listed in scrape_sources into hostdict ?  I can't seem to find it.

**Comment by jewbmx** [comment]
Thanks for both shares, i will have to check em out and see if i have any more vidsrc domains in my results that need added to it or whatnot.
As for bstsrs_one i had already checked and tested it but didnt need to change the decoder bit so i will need to go thru that again and likely combine our work.
And as for your question about the host domains thats sorta a mixed listing lol. some are from resolveurl, some are for url.replace stuff, and some are for our custom scraping. Dont think i ever had a need to send em into the hostdict list but if your wanting to the easiest place would likely be in the sources module either def getConstants or def getHostDict.

**Comment by jayshomebrew** [comment]
Just a heads up, the vidsrc sources were working for me using goojara_to.  I haven't tested other scrapers, so I doubt those will work.

**Comment by jewbmx** [comment]
Ok i tossed in your vidsrc stuff and prepared it for my testing while commenting out my old version incase its ever needed later.(packrat/hoarder style lol)

As for bstsrs_one ive checked all the domains you and i have in our 2 versions of the scraper and heres what i found...
bstsrs{dot}one is dead (no code to compare.)
bstsrs{dot}in is working (works in my scraper.)
topsrs{dot}day is working (works in your scraper.)
srstop{dot}link is working (works in your scraper.)

So now a question for ya/yall would you rather have me combine it all into one scraper or would you like to have me split them into 2 scrapers?
From my checking/testing the only real difference is domain name and the decoder used.(aka my scraper would be decoder v1 while your scrapers decoder v2 lol)
Also since our code in the scrapers is different we might need to decide which scraper version to use from now on since yours seems to account for some errors that i might not of noticed when using mine. Altho if both scrapers are used in the addon atleast 1 should always return results lol.(like mines named bstsrs and yours is named topsrs or srstop lol.)

**Comment by jayshomebrew** [comment]
Just use bstsrs_in and we'll keep the other in reserve.

**Comment by jewbmx** [comment]
K i set it up kinda decent and added all the domains then added a little ghetto domain check to control which decoder is used and added the new one as decoderV2 and renamed the old one as decoderV1.
Think i might of forgot something while doing all that tho so i might need to go back later and compare our scrapers again to see what i might of forgot lol.

Oh i also already found out that the new vidsrc code ya tossed to me didnt work on a scraper i was working on today, think it was a .me source link that redirected itself to be .net  which was ran thru with total failure but didnt look into that yet either. Figured i will bother once i get to the resolver end of the update work.
still got like 30+ scrapers to look at then still gonna check out that temprepo guys scraper piles i have and eventually snag a new url list to look thru for new scrapers to make lol.

I also need to hurry up a bit because finally after like a year or more of anime only i can swap back to normal movies and tv shows since there are some new kick ass movies out that i been waiting forever for :)

**Comment by jewbmx** [comment]
Got 2 questions for ya jayshomebrew lol, do you wanna fix the levidia_ch scraper since its similar to goojara_to which i think you fixed not long ago.
The other question was do ya wanna check out the sources from like 123moviestv_me?
they recently changed the api version again that most those sorta sites use so for like movies the servers_url is now '/ajax/episode/list/%s'
And the get_link url is '/ajax/episode/sources/%s'

The real issue im curious about in those "ajax" scrapers is if the source links are all duds or maybe i forgot a step lol.
The final source url i get is like  streameeeeee{dot}site/embed......
Trying any of them results always seems to be a file not found error and vidcloud for me lol altho i thought vidcloud was dead and gone too lmao

**Comment by jewbmx** [comment]
i went ahead and ghetto rigged both scrapers together the tweaked it a bit lol, you can check it out and see if its worth a remake or use this one i did and anything that might need fixed in this one too if its used lol.
https://github.com/jewbmx/ScraperWork/blob/main/temp/levidia_ch.py

**Comment by jayshomebrew** [comment]
Your levidia_ch scraper is working for me.  I'll check out 123movies later, but as I recall,  I also got stuck on those streameeeeee links.

**Comment by jewbmx** [comment]
I checked them on the browser and they work their so i figured it had to be something like a cookie or header thing so i added the usual self.cookie crap all the way thru the scraper then added a html scrape for them links to be ran thru and logged but now stumped on what needs done next lmao and the pages available arent easy to read so i think im stuck.
Probably a issue that gujal could handle and figure out but doesnt seem like hes active much lately so it might have to wait a while.

**Comment by jayshomebrew** [comment]
I got this older one to work; not great, but its providing links.  Give it a try when you get a chance:  holly moviehd cc

Added this one to test:  naija  vault com

I did spend some time looking at those streameeee links/123movies and was unsuccessful.

**Comment by jayshomebrew** [comment]
@jewbmx Can we squeeze in removing dups from aliases in sources_py?  I noticed some urls are kinda massive, +800k characters.

approx lines 571 & 583 [sources_py] 
```
seen = set()
aliases[:] = [d for d in aliases if (t := tuple(sorted(d.items()))) not in seen and not seen.add(t)]
```

**Comment by jayshomebrew** [comment]
@jewbmx so... I'm still working on few more scrapers. Is there still time to get them into the next build? #

**Comment by jewbmx** [comment]
Yea i havent finished my work either, you can just modify the scraper repo and fork it back if ya wanna share em or whatever

**Comment by jayshomebrew** [comment]
@jewbmx  alrighty. Put a bunch of pull requests for you in [scraperwork](https://github.com/jewbmx/ScraperWork/pulls).

**Comment by jewbmx** [comment]
K if your done adding em now i will grab em and start adding em into the addon along with your previous stuff.
I still plan to make a new url list and create new scrapers with it but ive put that idea on hold so the addon can be fixed up quicker.
Now all thats really left to do is to start fixing the final links we get like them random source links that are embed sites. I will make a list of the ones i have saved and share it here incase anyone wants to tinker with em too. Think once i get em all figured out the addon will be good enough to update and will work well again.

**Comment by Melann956** [comment]
Cannot wait. Thanks for all your hard work. ✌️❤️

________________________________

**Comment by jayshomebrew** [comment]
FYI: resolveURL just fixed luvuvid links from goojara, so thats good news.   

Any chance of adding a feature like this ?  URL list would be something like https://raw.githubusercontent.com/mylist.txt and the list would include URLs for custom scrapers. Just a thought.

<img width="963" height="417" alt="Image" src="https://github.com/user-attachments/assets/dc854e8a-e421-40a0-8fd8-c71588dc5864" />

**Comment by jewbmx** [comment]
Ive thought about setting something similar to that idea up but not sure how easy it would be and not sure if the normal imports for modules used would work either with the idea. My idea was more for adding a url to a zip of scrapers then unzipping it into like the userdata folder for use.

**Comment by jayshomebrew** [comment]
Got this working.  [zip of changes](https://github.com/jayshomebrew/ScraperWork/blob/dev_sources/new/plugin.video.scrubsv2.devscrapers.zip) to scrubsv2 that includes this update.

<img width="700" height="329" alt="Image" src="https://github.com/user-attachments/assets/c5ddf8b7-72f3-4292-ab8c-bb733a3f3383" /><BR> .zip file setting<BR>
---
<img width="700" height="412" alt="Image" src="https://github.com/user-attachments/assets/d19561bb-f561-4a6a-97a5-eb428c923753" /><BR>Installer<BR>
---

<img width="700" height="442" alt="Image" src="https://github.com/user-attachments/assets/c7ad2d2a-1f4a-4d46-b69a-b612f6b77000" /><BR>
---
<img width="700" height="481" alt="Image" src="https://github.com/user-attachments/assets/1759be93-c470-497b-892e-ba519980561b" /><BR>py files extracted<BR>

**Comment by jayshomebrew** [comment]
Added the option to delete the .py files in the devsources folder. 
<img width="700" height="473" alt="Image" src="https://github.com/user-attachments/assets/1374c65b-dadd-41c5-9c11-8139ff2868bc" />

**Comment by jewbmx** [comment]
I grabbed your stuff today and looked at it a little, at the moment i got 2 notes about it all lol. One is that your settings file didnt have the path setting which is seen in the userdata settings file and the actual code. Second is that all the code you threw in could probably just be made into a single module instead of being in a indexer.  Might take me a little while to go thru it all and test it out but it will probably be in the next update whenever i get motivated enough to dip back into coding lol

**Comment by jayshomebrew** [comment]
Added option shown below. [zip of changes](https://github.com/jayshomebrew/ScraperWork/blob/dev_sources/new/plugin.video.scrubsv2.devscrapers.zip)

I used a single foldername so that zip files were extracted to the sources folder.  Then these scrapers are auto-magically found without any other changes to scrubsv2. 

<img width="700" height="334" alt="Image" src="https://github.com/user-attachments/assets/91671729-e4d0-48c0-9c5b-a514e1747b4e" />
